### PR TITLE
docker configuration

### PR DESCRIPTION
**Pull request by ovv**

### CODE REVIEW

**Comment by ovv** [comment]
I'm open to continue working on it. Looks like there is a bunch of conflict now. I'll try to take a look soon at what was merged and rebase on that.

- why the names of the roles in the playbook?
I don't know, the guy from RedHat at the sprints said it was the recommended way to do it :man_shrugging:

**Comment by alexsavio** [comment]
Can I suggest you to just add the Github keys, the docker setup to clone the external repositories, and the names in the playbooks? I've done a few refactorings in #12, ready to get these features from you. Thanks a lot!

**Comment by ovv** [comment]
@alexsavio I have rebased on master. Please review again :)

File: README.md [hunk 1]
```diff
@@ -15,6 +15,16 @@ How to run this Ansible playbook
 pip install -r requirements.txt
 ```
 
+3. Launch the ansible playbook
+
+```bash
+ansible-playbook playbook.yml
```
Comment (webknjaz) [inline README.md]: I'd give the playbook a meaningful name

File: playbook.yml [hunk 2]
```diff
@@ -6,5 +6,5 @@
   roles:
     - { role: common, become: yes, become_method: sudo, tags: common }
     - { role: python, become: yes, become_method: sudo, tags: python }
-    - { role: docker-compose, tags: docker_compose }
+    - { role: docker-apps, tags: docker_apps }
```
[the remark this answers is not in the document]
Comment (webknjaz) [reply playbook.yml]: Yes. But maybe call them `docker-platform` and `web-apps`?

File: README.md (hunk 1 above)
Comment (alexsavio) [reply README.md]: what are meaninful names for playbooks? :) I usually call it playbook.yml

Comment (webknjaz) [reply README.md]: Do you call classes in Python just `Class`?

Comment (alexsavio) [reply README.md]: If I have only one in my whole project, most likely yes.

Comment (webknjaz) [reply README.md]: Don't. Also, you have different flows here. You have platform provisioning/bootstrapping and launching apps.

**Comment by alexsavio** [comment]
Since most of the features here look good to me, I will squash/merge this PR to keep things going, alright? We can work on the comments from @webknjaz in the following PRs

**Pull request merged**